src/main/java/de/szut/lf8_project/lf8/project/ProjectService.java

```diff
@@ -28,12 +31,40 @@ public ProjectEntity readById(long id) {
         }
         return queryedProject.get();
     }
+    public void deleteProjectById(long projectId, User currentUser) {
```

Comment:
What is user for ? and where is user comming from ?

Reply:
Wait i see it now why is the readById function replaced by your new function, readById should not have been touched.